setup.py

```diff
@@ -29,5 +29,6 @@
         'postgres': ['psycopg2', 'SQLAlchemy'],
         'mysql': ['mysql-python', 'SQLAlchemy'],
         'azure': ['azure'],
+        'xml': ['dicttoxml'],
```

Comment:
oh, `dicttoxml`  is also not imported lazily yet -- perhaps you can do that in xml_export_formatter.py too? :)